types/pdfkit/index.d.ts

```diff
@@ -332,6 +577,7 @@ declare namespace PDFKit.Mixins {
         alt?: string;
         expanded?: string;
         actual?: string;
+        title?: string;
```

Comment:
Here is the title type that I've added, via the `Structure Element Options` section of this page https://pdfkit.org/docs/accessibility.html

Reply:
This seems to be in the wrong interface, according to https://pdfkit.org/docs/accessibility.html#marked_content_options there is no title field in here.

Reply:
To me it looks like the problem is on line 321, that method shouldn't use the MarkingOptions interface. Instead, we should create a separate StructureElementOptions interface as described in https://pdfkit.org/docs/accessibility.html#structure_element_options